src/routes/Devilstable.svelte

```diff
@@ -71,45 +64,101 @@
 		}
 		return fractionData;
 	});
+	/** @type {{label: string, values: boolean[]}[]} */
 	let boolData = $derived([
-		...noFractions.map((d) => {
-			/** @type {boolean[]} */ const values = new Array(DATA_MAX).fill(false);
+		...inputChipValueLabels.map((d) => {
+			const info = data.find((c) => c.label === d);
+			if (info) {
+				/** @type {boolean[]} */ const values = new Array(DATA_MAX).fill(false);
+				info?.values.forEach(([start, end]) => {
+					for (let i = start; i <= end; i++) {
+						// Adjust for 0-indexed array
+						values[i - 1] = true;
+					}
+				});
 
-			d.values.forEach(([start, end]) => {
-				for (let i = start; i <= end; i++) {
-					// Adjust for 0-indexed array
-					values[i - 1] = true;
+				return {
+					label: d,
+					values
+				};
+			} else {
+				if (d === 'fr') {
+					return allFractionData;
+				} else {
+					return {
+						label: d,
+						values: new Array(DATA_MAX).fill(true)
+					};
 				}
-			});
-
-			return {
-				label: d.label,
-				values
-			};
-		}),
-		fractionData
+			}
+		})
 	]);
+	let selection = $state({
+		start: 1,
+		end: 100
+	});
+	let detailData = $derived(
+		boolData.map((d) => {
+			return { label: d.label, values: d.values.slice(selection.start - 1, selection.end) };
+		})
+	);
 </script>
 
+<div class="card p-4 variant-filled-primary max-w-lg" data-popup="popupClick">
```

Comment:
consider renaming 'popupClick' to something more specific 'popupChips' or 'popupChipsOnClick'